src/core/base/model.ts

```diff
@@ -33,6 +33,9 @@ export class UIPStateModel extends Observable {
   /** Last {@link UIPPlugin} element which changed markup. */
   private _lastModifier: UIPPlugin;
 
+  private _snippets: HTMLTemplateElement[];
```

Comment:
Consider using the script element instead.
HTMLTemplate is not supported by IE11.
PS: IE11 can be supported partially, but it should be supported